src/css/custom.scss

```diff
@@ -1,3 +1,8 @@
+/* https://dev.to/sajclarke_62/using-tailwindcss-v3-in-docusaurus-in-5-steps-5c26 */
```

Comment:
I'm not sure what the specific latest design is for text links and hover states, but we lost a lot of affordance here. There are a lot of text links throughout that you would probably miss now...

#### Inline content:
![Screenshot 2024-01-08 at 10 41 17 AM](https://github.com/cypress-io/cypress-documentation/assets/6298501/04a02012-49c9-4c42-b365-1e450a01ffb5)

#### "See also" lists:
![Screenshot 2024-01-08 at 12 02 42 PM](https://github.com/cypress-io/cypress-documentation/assets/6298501/4ff38b52-68d6-496c-82f8-0d282cdaf8c9)

Reply:
Good catch. We should probably make sure that links still look like links.

Reply:
fixed by [c1403e3](https://github.com/cypress-io/cypress-documentation/pull/5572/commits/c1403e3864be5ce7ecddbe18b82ad8b500772473)